Cargo.toml

```diff
@@ -27,8 +27,8 @@ default = ["std"]
 std = []
 # Reduce code size at the cost of performance.
 compact = []
-# Do not use a system allocator anywhere.
-no_alloc = []
+# Use the system allocator.
```

Comment:
This needs an additional minor version increment. The version should go to `0.2`, due to backwards-incompatible API changes.

Reply:
I agree.

Reply:
Push the relevant commit changing the version (you can merge the commits or not, either is fine) and I'll approve and merge the PR.